### DISCUSSION TITLE
Welcome to rotating_display Discussions!

### DISCUSSION THREAD

**Discussion by lhm0** [lhm0/rotating_display#1] [Announcements]
## 👋 Welcome!
Dear all,
 
please feel free to comment or ask questions regarding the Rotating Display project.

I am looking forward to your thoughts

Ludwin

**Comment by lhm0** [comment]
Hi Tomas,I am using something like this:EA1019HHES1201 - Steckernetzteil, 24 W, 12 V, 2 Areichelt.de12V, 2A. If you are interested in a rotating display, make sure to check out the latest version, which I have just published:lhm0/rotating_display_pico: A persistence of vision rotating display controlled by a Raspberry Pi Picogithub.comHere is a new video:Rotating Display pico - a persistence of vision display controlled by a Raspberry Pi Pico. DIY.https://youtu.be/Msv1gF6FXegAm 05.11.2024 um 06:45 schrieb GenTomasBM ***@***.***>:﻿
Hi, what power supply exactly are you using?

**Comment by lhm0** [comment]
2A is enough, 3A will work as wellAm 05.11.2024 um 07:52 schrieb GenTomasBM ***@***.***>:﻿
So 2A is enough? Can I use a 3A one?

**Comment by lhm0** [comment]
You mean in KiCad? Unfortunately, it needs a bypass solution, as KiCAD does not have circle pcb tracks. I approximated the circles with multiple straight lines. Start and endpoint coordinates were calculated with a little  helper program and copied manually into the KiCAD pcb file. That is easy, as it is a simple text file.Am 14.11.2024 um 07:59 schrieb GenTomasBM ***@***.***>:﻿
Hi, i wanted to ask how did you do the coil?

**Comment by lhm0** [comment]
The theoretical framework is described very well in the Mikrocontroller.net article. You don‘t need more.

I started with copper wire coils and a bread board setup to understand the circuit in depth (how to make a proper bifilar coil, how many windings for the feedback coil, the correct polarity, etc.). Then I used this online calculator to properly design a flat printed coil with the desired inductance:

http://www.circuits.dk/calculator_flat_spiral_coil_inductor.htm

There was quite some „trial and error“ involved, but even the first PCB version worked.

It is quite remarkable, that the circuit works in a very wide voltage range. I tested supply voltages of 5 to 24 V - no problem. At 24V it needs high power transistors, but the coil is the same.

I also experimented on improving the efficiency. The efficiency of the circuit is limited by the diameter of the coil. If you make it larger, you could use wider traces for the coil and thus improve the quality factor of the resonant circuit. I also thought about making a double sided coil, but I have not looked into this idea, yet.

I trust that helps.

Am 26.11.2024 um 05:40 schrieb GenTomasBM ***@***.***>:﻿
thanks, but i still dont understand how di you make the coil to work, like how did you connect it to the components in that way, is there any page that you used? I looked at the microkontroller.net but it didnt hlep me much

**Comment by GenTomasBM** [comment]


**Reply by lhm0** [reply]
All PCBs have standard thickness of 1.6 mm.

**Comment by GenTomasBM** [comment]


**Reply by lhm0** [reply]
I have no idea what the "A" stands for. However, the electrical specification of the RF-300FA-12350 seems to be the same. Make sure, your motor has the CD tray. I have seen dealers selling motors without tray.

**Comment by GenTomasBM** [comment]
Hey, is it possible to contact you in any other way? I would like to send you my pcb, and if you could try to help me, because I have a problem with the converter, the tranzistors right away start to burn.

**Reply by lhm0** [reply]
Hi Tomas, 

you can send me a private message on instructables.com. We can exchange email addresses there.

This is the project description page: https://www.instructables.com/Rotating-LED-Display/

**Reply by GenTomasBM** [reply]
Have you gotten the email?

**Reply by lhm0** [reply]
No, sorry. How did you send it?

**Reply by GenTomasBM** [reply]
I wrote you on instructables

**Reply by GenTomasBM** [reply]
I can't register there, because I need a German number

**Reply by GenTomasBM** [reply]
Maybe if you would send a direct link, it could work. And I did send you a direct message on instructables
